### ISSUE TITLE
ALLOWED_SUBNET="0.0.0.0/0"

### ISSUE THREAD

**Issue by errolsancaktar** [HaschekSolutions/PictShare-Docker#19]
Just a heads up i noticed an interesting issue where pictshare does not like a blank value for allowed_subnet anymore (at least with the docker version.  I have added the line i used to solve from my docker-compose.yaml:
      - ALLOWED_SUBNET="0.0.0.0/0"

I tried to wrap the block in an if statement in pictshare.sh but it also doesnt like an unassigned variable.

**Comment by geek-at** [comment]
Yes there was a bug in Pictshare that took empty ALLOWED_SUBNET configs for "nobody can view it". [Was fixed in the last commit](https://github.com/HaschekSolutions/pictshare/commit/01193683760d38d04002c1f1527ef7683f4a6040) though
